pkg/manifestclient/write_roundtripper.go

```diff
@@ -28,18 +28,21 @@ type writeTrackingRoundTripper struct {
 	// requestInfoResolver is the same type constructed the same way as the kube-apiserver
 	requestInfoResolver *apirequest.RequestInfoFactory
 
+	getKindForResourceFunc func(gvr schema.GroupVersionResource) (kindData, error)
+
 	lock              sync.RWMutex
 	nextRequestNumber int
 	actionTracker     *AllActionsTracker[TrackedSerializedRequest]
 }
 
-func newWriteRoundTripper() *writeTrackingRoundTripper {
+func newWriteRoundTripper(getKindForResourceFunc func(gvr schema.GroupVersionResource) (kindData, error)) *writeTrackingRoundTripper {
```

Comment:
don't pass a function here.  just hardcode the right thing.

Reply:
done, ptal.